### ISSUE TITLE
Bug: `clsPerformance` does not clear out last Operation 

### ISSUE THREAD

**Issue by hecon5** [joyfullservice/msaccess-vcs-addin#331]
The performance class doesn't clear `this.OperationName` if there is only one item in the call stack when ending operation timing.

Also, fixed an issue if the category or operation is longer than 30 characters, see draft.

PR to come.

```
debug.Print perf.GetReports
--------------------------------------------------
                PERFORMANCE REPORTS
--------------------------------------------------
Category                      Count     Seconds
--------------------------------------------------
Short Category                1         6.83
Long Category name from long  1         1.14
    town
--------------------------------------------------
TOTALS:                       2         7.98
--------------------------------------------------

--------------------------------------------------
Operations                    Count     Seconds
--------------------------------------------------
modStringTools.FitStringToCol 64        2.00
    umn
modStringTools.ListResultInde 60        17.84
    nt
--------------------------------------------------

```

**Comment by hecon5** [comment]
The PR will also include the option to get a JSON output of the performance class items

**Comment by hecon5** [comment]
Fixed by #441
